Regression/WarpX-tests.ini

```
@@ -1271,6 +1271,21 @@ tolerance = 1.e-14
 buildDir = .
 inputFile = Examples/Tests/TwoParticle_electrostatic/inputs_3d
 dim = 3
+addToCompileString = DO_ELECTROSTATIC=TRUE
```

Is this flag still needed or already ignored?
```suggestion
addToCompileString =
```

---

I agree. Actually the line `addToCompileString` could be remove altogether.